pkg/api/grafana_com_proxy.go

```diff
@@ -14,8 +13,7 @@ import (
 )
 
 var grafanaComProxyTransport = &http.Transport{
-	TLSClientConfig: &tls.Config{InsecureSkipVerify: false},
-	Proxy:           http.ProxyFromEnvironment,
+	Proxy: http.ProxyFromEnvironment,
```

Comment:
Presumably this doesn't need to be user-configurable as `Grafana.com` should always have a valid certificate.

Reply:
Yes.